README.md

```diff
@@ -1,43 +1,45 @@
-Azure Toolkits for Java is an open-source project that helps Java developers easily create, develop, configure, test, and deploy highly available and scalable Java web apps to Azure from [Eclipse](https://docs.microsoft.com/en-us/java/azure/eclipse/azure-toolkit-for-eclipse) and [IntelliJ IDEA](https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij) on all supported platforms. 
-* [Azure Toolkit for IntelliJ IDEA](https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij)
-* [Azure Toolkit for Eclipse](https://docs.microsoft.com/en-us/java/azure/eclipse/azure-toolkit-for-eclipse)
-* [Release Notes](https://github.com/Microsoft/azure-tools-for-java/releases)
-* [Issues](https://github.com/Microsoft/azure-tools-for-java/issues)
+[![official JetBrains project](http://jb.gg/badges/official-flat-square.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 
-## Azure Toolkit for IntelliJ IDEA
+# Azure plugin for JetBrains Rider [![Teamcity](http://teamcity.jetbrains.com/app/rest/builds/buildType:(id:AzureToolsForIntellij_Build)/statusIcon)](http://teamcity.jetbrains.com/viewType.html?buildTypeId=AzureToolsForIntellij_Build&guest=1)
 
-### Installation
-* [Set up the toolkits for IntelliJ](https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-installation)
-* [IntelliJ IDEA Plugin Repository](https://plugins.jetbrains.com/plugin/8053?pr=idea)
+The Azure plugin for [JetBrains Rider](https://www.jetbrains.com/rider) is an open-source project that helps .NET developers easily create, develop, configure, test, manage and deploy highly available and scalable web apps to Azure.
 
-### Documentation 
-* [Get Started Tutorial](https://docs.microsoft.com/en-us/azure/app-service-web/app-service-web-intellij-create-hello-world-web-app)
-* [Home Page of Azure Toolkit for IntelliJ](https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij)
-* [Java Developer Center on Azure](https://docs.microsoft.com/en-us/java/azure/)
-* [Get Started for HDInsight](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-apache-spark-intellij-tool-plugin)
+## Functionality
 
-## Azure Toolkit for Eclipse
+* Sign in to an Azure account
+* Manage one or multiple subscriptions
+* Basic management of container hosts (create/delete, start/stop, view details)
+* List container registries
+* Basic management of Redis caches (create/delete, start/stop, view details, list keys/values)
+* Basic management of storage accounts (create/delete, list/create/delete blob container, list/upload/download/delete blobs)
+* Basic management of virtual machines (create/delete, start/stop, view details)
+* Basic management of web apps (create/delete, start/stop, view details)
+* Run configuration to deploy ASP.NET Core web app (can also provision SQL database)
```

Comment:
We do support Core web apps on any platform and full .Net framework web apps on Windows only.